addon/templates/components/flexberry-edit-layermap.hbs

```diff
@@ -72,6 +72,19 @@
     </div>
     <div class="field">
       <label>
+        {{scaleTextboxCaption}}
```

Comment:
Is scale required for group layers, @kuzkok ?

Reply:
no